### DISCUSSION TITLE
Using forked Polygon node in test environment

### DISCUSSION THREAD

**Discussion by piotrkorgol** [Web3Auth/Web3Auth#1078] [Help]
Hello,

I want to use a forked Polygon node in our test environment with Hardhat. Will Web3Auth work with that out of the box?

Kind regards

**Comment by himanshuchawla009** [comment]
Yes, you can pass you custom node url in chainConfig along with chainId, as long as the node is evm compatible (exposes same API as polygon, it will work seamlessly)
